partiql-planner/src/test/kotlin/org/partiql/planner/util/PlanNodeEquivalentVisitor.kt

```diff
@@ -125,13 +125,6 @@ class PlanNodeEquivalentVisitor : PlanBaseVisitor<Boolean, PlanNode>() {
         return true
     }
 
-    override fun visitRelOpExcludeStepCollIndex(node: Rel.Op.Exclude.Step.CollIndex, ctx: PlanNode): Boolean {
```

Comment:
This change from the `main` merge to `partiql-eval` commit da03142909b808486e0f170adb52f55d0c0c1eb5 also caused a build failure on `partiql-eval`. `EXCLUDE`'s modeling on `partiql-eval` was changed so this visitor function was not overriding anything.

Reply:
Cool. I'll rebase